app-android/src/main/java/com/ustadmobile/port/android/view/PersonEditFragment.kt

```diff
@@ -222,6 +222,24 @@ class PersonEditFragment: UstadEditFragment<PersonWithAccount>(), PersonEditView
             handleInputError(mBinding?.usernameTextinputlayout, value != null, value)
         }
 
+    override var firstNamesFieldError: String? = null
```

Comment:
These can be handled on the view XML as they are on other views. I think we don't need the handleInputError function.